src/course-unit/course-xblock/CourseXBlock.test.jsx

```diff
@@ -217,4 +245,54 @@ describe('<CourseXBlock />', () => {
       expect(handleConfigureSubmitMock).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('edit', () => {
+    it('navigates to editor page on edit HTML xblock', async () => {
+      const { getByText, getByLabelText } = renderComponent({
+        type: COMPONENT_ICON_TYPES.html,
+      });
+
+      await waitFor(() => {
```

Comment:
[important]: Let's check if `waitFor` is needed in all the tests in this file.

[optional]: Also in some tests in this file, you looked for buttons using `getByText`, I suggest finding them using `getByRole`

Reply:
Removed `waitFor` and changed `getByLabelText` to `getByRole`